### DISCUSSION TITLE
Frigate Version in Docker?

### DISCUSSION THREAD

**Discussion by dz4news** [blakeblackshear/frigate#4626] [Ask A Question] [answered]
I am running frigate in a docker container. I used Docker Compose to spin everything up and it is working very well. But how do I know I am running the latest version of Frigate?
I used the code below (tuncated) in the compose file to create the container... so is 3.9 the version or does it just use the latest version at the time of creation?
I am very new to docker... so I could be totally missing something..

`version: "3.9"
services:
  frigate:
    container_name: frigate
    privileged: true # this may not be necessary for all setups
    restart: unless-stopped
    image: blakeblackshear/frigate:stable
    shm_size: "64mb" # update for your cameras based on calculation above`

**Comment by spacebares** [comment, marked as answer]
3.9 is the docker-compose file version.
once frigate is up you can click debug / system on the left panel of the website and frigate version should be shown on that page.
